### ISSUE TITLE
FEATURE - Update packages.yml to allow for dbt-utils 0.7.0

### ISSUE THREAD

**Issue by mvtemple** [fivetran/dbt_stripe_source#17]
**Are you a Fivetran customer?** 
Meera, Consultant, Ephemeral Tattoos

**Is your feature request related to a problem? Please describe.**
I'm trying to use dbt-utils 0.7.0 (the latest release) so that I can use dbt 0.20.0 with my M1 mac, but this is conflicting with the dbt-utils version requirement in my Fivetran package

**Describe the solution you'd like**
Change the version check to allow for dbt-utils 0.7.0

**Describe alternatives you've considered**
 

**Additional context**

**Please indicate the level of urgency and business impact of this request** 

**Are you interested in contributing to this package?** 

- [ ] Yes, I can do this and open a PR for your review.
- [ ] Possibly, but I'm not quite sure how to do this. I'd be happy to do a live coding session with someone to get this work implemented. 
- [X] No, I'd prefer if someone else did this.  I don't have the time and/or don't know how to incorporate the changes necessary.

**Comment by fivetran-joemarkiewicz** [comment]
Hi @mvtemple 👋 

Thanks for opening this Issue. We are currently in the process of scoping out the efforts across our packages and fivetran_utils to be compatibile with dbt 0.20.0 and the new dbt_utils v0.7.0. The upgrade nature will most likely take a while as we want to ensure our packages and previous releases will work properly once we upgrade. As such, we will most likely not be able to address this feature request for a few weeks.

In the meantime, you could possibly try to add `--no-version-check` to your dbt commands. This may resolve the version error you are currently seeing. Otherwise, I will continue to track updates to this Issue here so you may be alerted when we upgrade our packages.

Thank you!

**Comment by fivetran-joemarkiewicz** [comment]
Hi @mvtemple Just wanted to give you an update that we will be pushing dbt v0.20.0 upgrades to our packages (including this one) next week. I will post back here once these changes are live. 

Thank you for your patience!

**Comment by mvtemple** [comment]
Hey @fivetran-joemarkiewicz thanks for this update! Have the changes gone through as yet?

**Comment by fivetran-joemarkiewicz** [comment]
Hey @mvtemple I am making the changes as we speak PR #23 ! Should be good to go for both the source and transform by the end of the day tomorrow!

**Comment by fivetran-joemarkiewicz** [comment]
@mvtemple Just wanted to give you an update that we have merged the PR and cut a new release which incorporates dbt 20 compatibility as well as referencing the updated dbt_utils. However, It looks like the [PR](https://github.com/dbt-labs/hub.getdbt.com/pull/744) which would make the release live on the dbt hub has not been merged yet. Once this PR is merged we will be able to cut the transform release and you will be good to go. 

I will keep an eye on the PR linked and will post back here once it is live on the hub! Have a great weekend 🎉

**Comment by mvtemple** [comment]
@fivetran-joemarkiewicz, appreciate the updates!!

**Comment by fivetran-joemarkiewicz** [comment]
@mvtemple We were able to cut the dbt 20 compatibility for stripe release `v0.5.0`! This is now live on the [dbt hub](https://hub.getdbt.com/fivetran/stripe/latest/)

Feel free to open another issue if you have any questions moving forward. Thanks!
